msg/MotionPlanRequest.msg

```diff
@@ -51,6 +51,21 @@ float64 allowed_planning_time
 float64 max_velocity_scaling_factor
 float64 max_acceleration_scaling_factor
 
+# New joint limits to use when running time-parameterization and smoothing.
+# Note, this does not replace the values in the robot model.
+# Only applicable for TOTG and Ruckig
```

Comment:
It should be applied to all parameterization steps

Reply:
I think we should re-visit deleting the deprecated parameterization algorithms, `AddTimeParameterization` and `AddIterativeSplineParameterization`. Anyway, I generally agree with Henning so I'd just delete this line.

```suggestion
```